src/olympia/stats/tests/test_views.py

```diff
@@ -279,36 +289,25 @@ def csv_eq(self, response, expected):
 
 
 class TestViews(ESStatsTestCase):
-    def _test_no_cache(self):
```

Comment:
I removed this test case. It wasn't running anyway and we have another test class for cache.